graylog2-web-interface/src/views/components/widgets/Widget.aggregations.test.tsx

```diff
@@ -47,10 +47,19 @@ import FieldTypesContext from '../contexts/FieldTypesContext';
 import ViewTypeContext from '../contexts/ViewTypeContext';
 
 const testTimeout = applyTimeoutMultiplier(30000);
+const mockedUnixTime = 1577836800000; // 2020-01-01 00:00:00.000
 
 jest.mock('./WidgetHeader', () => 'widget-header');
 jest.mock('./WidgetColorContext', () => ({ children }) => children);
 
+jest.mock('moment-timezone', () => {
+  const momentMock = jest.requireActual('moment-timezone');
+  momentMock.tz.setDefault('UTC');
+  momentMock.tz.guess = () => 'UTC';
+
+  return momentMock;
+});
```

Comment:
We need to define the timezone for moment here, because we are using `moment.tz.guess` in `DateTime.getBrowserTimezone`. Unfortunately we can not use `AppConfig.rootTimeZone()` in this case.

We should include this mock in our jest setup in a separate PR. In the same step we can implement a constant for   the root time zone and use it to mock `moment.tz` and `AppConfig.rootTimeZone()`